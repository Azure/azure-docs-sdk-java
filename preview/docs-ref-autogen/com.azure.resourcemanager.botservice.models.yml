### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.botservice.models"
fullName: "com.azure.resourcemanager.botservice.models"
name: "com.azure.resourcemanager.botservice.models"
summary: "Package containing the data models for AzureBotService. Azure Bot Service is a platform for creating smart\n conversational agents."
classes:
- "com.azure.resourcemanager.botservice.models.AlexaChannel"
- "com.azure.resourcemanager.botservice.models.AlexaChannelProperties"
- "com.azure.resourcemanager.botservice.models.BotProperties"
- "com.azure.resourcemanager.botservice.models.BotResponseList"
- "com.azure.resourcemanager.botservice.models.Channel"
- "com.azure.resourcemanager.botservice.models.ChannelResponseList"
- "com.azure.resourcemanager.botservice.models.ChannelSettings"
- "com.azure.resourcemanager.botservice.models.CheckNameAvailabilityRequestBody"
- "com.azure.resourcemanager.botservice.models.ConnectionSettingParameter"
- "com.azure.resourcemanager.botservice.models.ConnectionSettingProperties"
- "com.azure.resourcemanager.botservice.models.ConnectionSettingResponseList"
- "com.azure.resourcemanager.botservice.models.DirectLineChannel"
- "com.azure.resourcemanager.botservice.models.DirectLineChannelProperties"
- "com.azure.resourcemanager.botservice.models.DirectLineSite"
- "com.azure.resourcemanager.botservice.models.DirectLineSpeechChannel"
- "com.azure.resourcemanager.botservice.models.DirectLineSpeechChannelProperties"
- "com.azure.resourcemanager.botservice.models.EmailChannel"
- "com.azure.resourcemanager.botservice.models.EmailChannelProperties"
- "com.azure.resourcemanager.botservice.models.FacebookChannel"
- "com.azure.resourcemanager.botservice.models.FacebookChannelProperties"
- "com.azure.resourcemanager.botservice.models.FacebookPage"
- "com.azure.resourcemanager.botservice.models.KikChannel"
- "com.azure.resourcemanager.botservice.models.KikChannelProperties"
- "com.azure.resourcemanager.botservice.models.Kind"
- "com.azure.resourcemanager.botservice.models.LineChannel"
- "com.azure.resourcemanager.botservice.models.LineChannelProperties"
- "com.azure.resourcemanager.botservice.models.LineRegistration"
- "com.azure.resourcemanager.botservice.models.MsTeamsChannel"
- "com.azure.resourcemanager.botservice.models.MsTeamsChannelProperties"
- "com.azure.resourcemanager.botservice.models.MsaAppType"
- "com.azure.resourcemanager.botservice.models.OperationDisplayInfo"
- "com.azure.resourcemanager.botservice.models.OperationEntityListResult"
- "com.azure.resourcemanager.botservice.models.OperationResultStatus"
- "com.azure.resourcemanager.botservice.models.PrivateEndpoint"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.botservice.models.PrivateLinkResource"
- "com.azure.resourcemanager.botservice.models.PrivateLinkResourceBase"
- "com.azure.resourcemanager.botservice.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.botservice.models.PublicNetworkAccess"
- "com.azure.resourcemanager.botservice.models.ServiceProvider"
- "com.azure.resourcemanager.botservice.models.ServiceProviderParameter"
- "com.azure.resourcemanager.botservice.models.ServiceProviderParameterMetadata"
- "com.azure.resourcemanager.botservice.models.ServiceProviderParameterMetadataConstraints"
- "com.azure.resourcemanager.botservice.models.ServiceProviderProperties"
- "com.azure.resourcemanager.botservice.models.Site"
- "com.azure.resourcemanager.botservice.models.SiteInfo"
- "com.azure.resourcemanager.botservice.models.Sku"
- "com.azure.resourcemanager.botservice.models.SkuName"
- "com.azure.resourcemanager.botservice.models.SkuTier"
- "com.azure.resourcemanager.botservice.models.SkypeChannel"
- "com.azure.resourcemanager.botservice.models.SkypeChannelProperties"
- "com.azure.resourcemanager.botservice.models.SlackChannel"
- "com.azure.resourcemanager.botservice.models.SlackChannelProperties"
- "com.azure.resourcemanager.botservice.models.SmsChannel"
- "com.azure.resourcemanager.botservice.models.SmsChannelProperties"
- "com.azure.resourcemanager.botservice.models.TelegramChannel"
- "com.azure.resourcemanager.botservice.models.TelegramChannelProperties"
- "com.azure.resourcemanager.botservice.models.WebChatChannel"
- "com.azure.resourcemanager.botservice.models.WebChatChannelProperties"
- "com.azure.resourcemanager.botservice.models.WebChatSite"
enums:
- "com.azure.resourcemanager.botservice.models.ChannelName"
- "com.azure.resourcemanager.botservice.models.Key"
- "com.azure.resourcemanager.botservice.models.RegenerateKeysChannelName"
interfaces:
- "com.azure.resourcemanager.botservice.models.Bot"
- "com.azure.resourcemanager.botservice.models.Bot.Definition"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.Blank"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.WithEtag"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.WithKind"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.WithSku"
- "com.azure.resourcemanager.botservice.models.Bot.DefinitionStages.WithTags"
- "com.azure.resourcemanager.botservice.models.Bot.Update"
- "com.azure.resourcemanager.botservice.models.Bot.UpdateStages"
- "com.azure.resourcemanager.botservice.models.Bot.UpdateStages.WithEtag"
- "com.azure.resourcemanager.botservice.models.Bot.UpdateStages.WithKind"
- "com.azure.resourcemanager.botservice.models.Bot.UpdateStages.WithProperties"
- "com.azure.resourcemanager.botservice.models.Bot.UpdateStages.WithSku"
- "com.azure.resourcemanager.botservice.models.Bot.UpdateStages.WithTags"
- "com.azure.resourcemanager.botservice.models.BotChannel"
- "com.azure.resourcemanager.botservice.models.BotConnections"
- "com.azure.resourcemanager.botservice.models.Bots"
- "com.azure.resourcemanager.botservice.models.Channels"
- "com.azure.resourcemanager.botservice.models.CheckNameAvailabilityResponseBody"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.Definition"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.Blank"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.WithEtag"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.WithKind"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.WithSku"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.DefinitionStages.WithTags"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.Update"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.UpdateStages"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.UpdateStages.WithEtag"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.UpdateStages.WithKind"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.UpdateStages.WithProperties"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.UpdateStages.WithSku"
- "com.azure.resourcemanager.botservice.models.ConnectionSetting.UpdateStages.WithTags"
- "com.azure.resourcemanager.botservice.models.DirectLines"
- "com.azure.resourcemanager.botservice.models.HostSettings"
- "com.azure.resourcemanager.botservice.models.HostSettingsResponse"
- "com.azure.resourcemanager.botservice.models.ListChannelWithKeysResponse"
- "com.azure.resourcemanager.botservice.models.OperationEntity"
- "com.azure.resourcemanager.botservice.models.OperationResults"
- "com.azure.resourcemanager.botservice.models.OperationResultsDescription"
- "com.azure.resourcemanager.botservice.models.Operations"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnection.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnection.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.botservice.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.botservice.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.botservice.models.PrivateLinkResources"
- "com.azure.resourcemanager.botservice.models.ServiceProviderResponseList"
metadata: {}
package: "com.azure.resourcemanager.botservice.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-botservice:1.0.0-beta.4
