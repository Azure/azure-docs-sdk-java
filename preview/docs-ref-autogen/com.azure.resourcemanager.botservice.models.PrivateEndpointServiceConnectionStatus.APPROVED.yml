### YamlMime:JavaMember
uid: "com.azure.resourcemanager.botservice.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.botservice.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.botservice.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.botservice.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.botservice.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-botservice:1.0.0-beta.4
